CMakeLists.txt

```diff
@@ -474,6 +486,10 @@ if(CMAKE_COMPILER_IS_GNUCXX AND NOT APPLE)
       set(US_LINK_FLAGS "${US_LINK_FLAGS} ${_linkflag}")
     endif()
   endforeach()
+  if(ARM)
+    # atomic intrinsics are in libatomic on ARM GCC
+    set(US_LINK_FLAGS "${US_LINK_FLAGS} -latomic")
```

Comment:
Are there any other ARM compilers other than GCC? If so, do they have support for ```-latomic```?

If the intent is only to support ARM GCC compilers, that's fine with me, however I would prefer to have a stricter check for this on lines 114 - 117 and to reflect the restriction in the variable name (e.g. ```GCC_ARM``` instead of ```ARM```). This way other developers can understand the ARM toolchain supported should they want to add other ARM toolchain support.